src/Docker.DotNet/DockerClientConfiguration.cs

```diff
@@ -6,33 +6,37 @@ namespace Docker.DotNet
 {
     public class DockerClientConfiguration : IDisposable
     {
-        public Uri EndpointBaseUri { get; internal set; }
-
-        public Credentials Credentials { get; internal set; }
-
+	    public Uri EndpointBaseUri { get; internal set; }
+		
+		public Credentials Credentials { get; internal set; }
+		
         public TimeSpan DefaultTimeout { get; internal set; } = TimeSpan.FromSeconds(100);
-
+    
         public TimeSpan NamedPipeConnectTimeout { get; set; } = TimeSpan.FromMilliseconds(100);
 
-        private static Uri LocalDockerUri()
+		private static Uri LocalDockerUri()
```

Comment:
Confused by the spacing. Seems like these are true tabs?

Reply:
Spacing issues come from removing non functional changes. After this changeset is aproved I'll submit a new one applying 
Roslyn compiler recommendations and reordering procedures within files, with no functional changes.